fv3core/stencils/basic_operations.py

```diff
@@ -18,6 +18,17 @@ def copy_stencil(q_in: FloatField, q_out: FloatField):
         q_out = q_in
 
 
+def copy_defn(q_in: FloatField, q_out: FloatField):
```

Comment:
nice -- I did this same thing with copy_stencil_method in master now, but I like your name better, would you mind replacing that with this?

Reply:
done